src/components/TrendingSkeleton.tsx

```
@@ -0,0 +1,104 @@
+    <View className="py-4">
+      {/* Hashtags section skeleton */}
+      <View className="mb-6">
+        <Animated.View 
+          style={shimmerStyle}
+          className="h-4 bg-gray-700 rounded mb-3 w-32"
+        />
+        <View className="flex-row flex-wrap">
+          {[1, 2, 3, 4, 5].map((index) => (
+            <View key={index} className="mr-2 mb-2">
+              <Animated.View 
+                style={shimmerStyle}
```

Biome (2.1.2)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)